crates/isograph_lang_parser/src/parse_iso_literal.rs

```diff
@@ -7,7 +7,10 @@ use common_lang_types::{
 use graphql_lang_types::{
     ListTypeAnnotation, NamedTypeAnnotation, NonNullTypeAnnotation, TypeAnnotation,
 };
-use intern::string_key::{Intern, StringKey};
+use intern::{
+    string_key::{Intern, StringKey},
+    Lookup,
```

Comment:
No need for `Lookup` anymore

Reply:
Fixed with https://github.com/isographlabs/isograph/pull/4/commits/512fa229076b8a510e92dd1b1ad1e8a898b9afdf